circuits/credentialAtomicQueryV3.circom

```diff
@@ -20,7 +21,7 @@ component main{public [requestID,
                        value,
                        timestamp, 
                        isRevocationChecked,
-                       issuerClaimIdenState, // MTP specific
```

Comment:
why it is removed? I see no other state ? is it ouput?

Reply:
yes, issuerState is the new output